crates/ra_analysis/src/db.rs

```diff
@@ -105,6 +105,7 @@ salsa::database_storage! {
             fn type_for_field() for hir::db::TypeForFieldQuery;
             fn struct_data() for hir::db::StructDataQuery;
             fn enum_data() for hir::db::EnumDataQuery;
+            fn impls_in_crate() for hir::db::ImplsInCrateQuery;
```

Comment:
This seems to be dead code (repeated in mock as well).

Reply:
Oh, I forgot to rename the type for the query and so also to rename these...